tests/fixtures/helpers/reload_configuration_new_config_mp.yaml

```diff
@@ -0,0 +1,15 @@
+test_platform:
+  - test_domain:
+      - sensors:
+          combined_sensor_energy_usage:
+            friendly_name: Combined Sense Energy Usage
+            unit_of_measurement: kW
+            value_template:
+              "{{ ((states('sensor.energy_usage') | float) + (states('sensor.energy_usage_2')
+              | float)) / 1000 }}"
+          watching_tv_in_master_bedroom:
+            friendly_name: Watching TV in Master Bedroom
+            value_template:
+              '{% if state_attr("remote.alexander_master_bedroom","current_activity")
+              == "Watch TV" or state_attr("remote.alexander_master_bedroom","current_activity")
+              == "Watch Apple TV" %}on{% else %}off{% endif %}'
```

Comment:
We need a test with both the old and new way in the same test to make sure both can be reloaded. `rest` currently allows this

Reply:
Resolved in https://github.com/home-assistant/core/pull/93538/commits/3294ab9fed01863b0512db8264ebb4db7259f467